tests/e2e/framework/kubernetes.go

```diff
@@ -206,15 +213,26 @@ func newKubeInfo(tmpDir, runID, baseVersion string) (*KubeInfo, error) {
 
 	a := NewAppManager(tmpDir, *namespace, i, kubeConfig)
 
+	var l *LocalRegistry
```

Comment:
Why ? The makefile must install this before the test is run. So it is running, and can be cleand up ( if needed ) by the makefile. Why complicate the tests with this code ?